src/test/java/net/openhft/chronicle/wire/TestJsonIssue467.java

```diff
@@ -22,27 +22,26 @@
 import net.openhft.chronicle.bytes.OnHeapBytes;
 import org.jetbrains.annotations.NotNull;
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 
 import static net.openhft.chronicle.core.pool.ClassAliasPool.CLASS_ALIASES;
 
 // relates to https://github.com/OpenHFT/Chronicle-Wire/issues/467
 public class TestJsonIssue467 {
 
-    public static class ResponseItem extends SelfDescribingMarshallable {
+    public static class ResponseItem467 extends SelfDescribingMarshallable {
```

Comment:
The tests pass without renaming this inner class, however tests in JSONWithAMapTest do fail. 
JSONWithAMapTest also defines an inner class called ResponseItem and the tests fail with the following error:
"Did not replace by simpleName class net.openhft.chronicle.wire.TestJsonIssue467$ResponseItem with class net.openhft.chronicle.wire.marshallable.JSONWithAMapTest$ResponseItem"

I don't think it breaks the test to give TestJsonIssue467's inner class ResponseItem the suffix '467' to avoid this collision of class names and this seems like the simplest solution.